#layout
uint32 coeff_num
uint32 segment
float64 target
# data
float64[] coeff
float64[] time
